pkg/agent/cniserver/pod_configuration.go

```diff
@@ -446,40 +438,19 @@ func (pc *podConfigurator) reconcile(pods []corev1.Pod) error {
 		}
 	}
 
-	for pod := range desiredPods.Difference(actualPods) {
-		// This should not happen since OVSDB is persisted on the Node.
-		// TODO: is there anything else we should be doing? Assuming that the Pod's
-		// interface still exists, we can repair the interface store since we can
-		// retrieve the name of the host interface for the Pod by calling
-		// GenerateContainerInterfaceName. One thing we would not be able to
-		// retrieve is the container ID which is part of the container configuration
-		// we store in the cache, but this ID is not used for anything at the
-		// moment. However, if the interface does not exist, there is nothing we can
-		// do since we do not have the original CNI parameters.
-		klog.Warningf("Interface for Pod %s not found in the interface store", pod)
-	}
+	missedPods := desiredPods.Difference(actualPods)
+	pc.reconcileMissingPods(missedPods, containerAccess)
 	return nil
 }
 
-// connectInterfaceToOVS connects an existing interface to ovs br-int.
-func (pc *podConfigurator) connectInterfaceToOVS(
-	podName string,
-	podNameSpace string,
-	containerID string,
-	hostIface *current.Interface,
-	containerIface *current.Interface,
-	ips []*current.IPConfig,
-) (*interfacestore.InterfaceConfig, error) {
-	// Use the outer veth interface name as the OVS port name.
-	ovsPortName := hostIface.Name
-	containerConfig := buildContainerConfig(ovsPortName, containerID, podName, podNameSpace, containerIface, ips)
-
+func (pc *podConfigurator) connectInterfaceToOVSInternal(ovsPortName string, containerConfig *interfacestore.InterfaceConfig) error {
```

Comment:
How about connectInterfaceToOVSCommon?

Reply:
Thanks Jianjun. `connectInterfaceToOVSCommon` is better.